.changeset/silly-hotels-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+"vite-plugin-externalize-dependencies": patch
```

Comment:
Please change this to minor. Package is still technically in "pre-release" stage, although I will release a 1.0 soon.

Reply:
Done ✅